datasets/iowa_liquor_sales/docs/tutorials/liquor_sales_prediction/tutorial.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from datetime import datetime
</pre><p>I am not sure if you actually made use of datetime in your code. If not, flake8 will warn you and you can delete this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/public-datasets-pipelines/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>